reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
```

Comment:
It is acceptable only if you do the same style on all other pages, and then ensure that all future PRs and changes don't break the ordering, and finally tell users how to fix merge conflicts that would stem from it. :wink: